### ISSUE TITLE
Accessibility - Lighthouse: Buttons must have discernible text

### ISSUE THREAD

**Issue by cheney6110** [hackforla/food-oasis#2865]
### Overview
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

### Instructions/Resources
https://dequeuniversity.com/rules/axe/4.12/button-name
